### PR TITLE
Fix multimap deprecation for notebooks

### PR DESCRIPTION
**Pull request by DavidSlayback**
Pretty small PR, just replacing jax.tree_* calls with jax.tree_util.tree_* so that the notebooks work

### CODE REVIEW

**Comment by RobertTLange** [comment]
Thank you! I am pretty sure though that `jax.tree_map` is still supported though. I only believe multi_map was deprecated. Is there any obvious advantage to using `tree_utils`? See https://github.com/RobertTLange/gymnax/blob/66828360fe263c057340d014e637960fdc3a5e16/gymnax/environments/environment.py#L43-L45

**Comment by DavidSlayback** [comment]
Nope, sorry! I kept seeing deprecation warnings on all the other operations and assumed it would be applied to ALL tree operations. Makes more sense this way, tree_map is by far the most commonly-used function
